src/library_glfw.js

```diff
@@ -972,6 +972,27 @@ var LibraryGLFW = {
       Module.ctx = Browser.destroyContext(Module['canvas'], true, true);
     },
 
+    getScreenVideoMode: function() {
+      if (!GLFW.videoMode) {
+        GLFW.videoMode = allocate([0, 0, 0, 0, 0, 0], 'i32', ALLOC_NORMAL);
+      }
+
+      setValue(GLFW.videoMode + 0*4, window.screen.width, 'i32');
+      setValue(GLFW.videoMode + 1*4, window.screen.height, 'i32');
+
+      var redBits = window.screen.colorDepth / 3;
+      var greenBits = window.screen.colorDepth / 3;
+      var blueBits = window.screen.colorDepth / 3;
+      setValue(GLFW.videoMode + 2*4, redBits, 'i32');
+      setValue(GLFW.videoMode + 3*4, greenBits, 'i32');
+      setValue(GLFW.videoMode + 4*4, blueBits, 'i32');
+
+      var refreshRate = 60; // TODO: estimate from requestAnimationFrame?
```

Comment:
There is a pending todo on my table to write a `emscripten_get_requestanimationframe_rate()`, that can then be used here.